implementation/routing/src/routing_manager_impl.cpp

```diff
@@ -228,7 +228,19 @@ void routing_manager_impl::start() {
             << ", "
             << its_netmask_or_prefix.str();
 
-    netlink_connector_ = std::make_shared<netlink_connector>(
+    boost::asio::ip::address temp {configuration_->get_unicast_address()};
+    
+    // PRECONDITION: netlink_connector_ has troubles to create a correct object,
+    // if network interface is part of unicast-address-string. So strip it!
+    char delimiter {'%'}; // "%" is a symbol for the device, e.g. "beef::1%7",
```

Comment:
Is 7 a hardcoded value for the Network Interface you want to use?
If yes, we will need another solution not using an hardcoded value

Reply:
The feature is the "delimiter", not the value "7". The example string is for instrumental purposes so that other developers know, why the delimiter was chosen the way it is.